injection.js

```diff
@@ -409,7 +409,8 @@ function onWindowResize(event){
 
 function actuallyResizeWindow(event){
 	windowResizeId = null;
-	if(settings.iframeCheck == "on" || (settings.iframeCheck == "auto" && window.innerWidth > 500)){
+	if(settings.iframeCheck == "on" && location.pathname != "/map.html" || 
```

Comment:
Yeaaa that's weird. I think standalone map page doesn't need to use injection js as we don't need any of the guide checkbox stuff.

Reply:
I put it this way so that injection still generates the json data trees for location/nirnroot data.

Are we able to pull this data without running injection.js?

Reply:
yeah, the json data is loaded from some load function (I forget the name) in obliviondata.js
and the save data is loaded from some load function in progress.js

So we can just call those functions directly

Reply:
Done